lisa_commands/commands.py

```diff
@@ -3,36 +3,41 @@
 then gets the string to pass back to slack
 """
 
-import logging
 import random
 import re
 import json
 import request_helper as request
 
-def get_return_text(slack_event):
+def message_text(slack_event):
     """
-    Returns the response we are giving to slack
-    :raw_text the raw message text we received
-    :message_user the user who sent the message
-    :return: text we want to return to slack
+    Gets text to send back to slack
+
+    slack_event - a dictionary containing slack event information
+
+    Returns the message text we will send to Slack
     """
+
     _bot_user_id, message = parse_mention(slack_event['text'])
     return run_command(message, '', slack_event)
 
-def parse_mention(message_text):
+def parse_mention(text):
     """
     Finds direct mentions (mentions at the front of a message)
-    :param message_text: The message text
-    :return: User ID mentioned, or if no direct mention, None
+
+    text - a dictionary containing slack event information
+
+    Returns an 2 element array of [USER_ID, rest_of_message]
```

Comment:
nit: technically returns a [tuple](https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences) not an array.

Reply:
Good catch. I fixed it